packages/opentelemetry-exporter-zipkin/src/types.ts

```diff
@@ -20,12 +20,13 @@ import { ExportResult } from '@opentelemetry/core';
  * Exporter config
  */
 export interface ExporterConfig {
-  headers?: { [key: string]: string };
+  headers?: Record<string, string>;
   serviceName?: string;
   url?: string;
   // Optional mapping overrides for OpenTelemetry status code and description.
   statusCodeTagName?: string;
   statusDescriptionTagName?: string;
+  getHeadersBeforeSend?: () => Record<string, string> | undefined;
```

Comment:
I think export would be more clear than send for the name.

Reply:
not sure if I understand, can you propose the whole name ?

Reply:
In this case it would be `getHeadersBeforeExport`, but I actually think a name like `getExportRequestHeaders` would be more clear.

Reply:
It's minor. If you like your name better feel free to keep it. Everything else looks good

Reply:
will change it, finding a good name is always fun :D

Reply:
`getHeadersBeforeExport` - seems to be more intuitive for me though ?

Reply:
I like `getExportRequestHeaders` because the term "Request Headers" is already understood by many people. The term "send" could mean a lot of things.

Reply:
fine with me